e2e/scripts/st_session_state.py

```diff
@@ -0,0 +1,24 @@
+# Copyright 2018-2020 Streamlit Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import streamlit as st
+
+# st.session_state() can only run in streamlit
+if st._is_running_with_streamlit:
+    state = st.session_state(count=0)
```

Comment:
Ultimately it's just a dictionary but I think there could be a lot more test cases we could do. Ideally in a unit tests in favor of speed but yea...

Reply:
That is true, though there are more bugs associated by bringing in this functionality, hence a solution for callbacks. Those callbacks will have more e2e tests.